### ISSUE TITLE
O...M...G.... THANK YOU

### ISSUE THREAD

**Issue by Skrell** [michaelxzhang/npp-highlighter#25]
I've been a VERY LONG time user of Notepad++ and this plugin gives me THE feature I had wished for the most!!!  TYTYTYTY

**Comment by shodanx2** [comment]
I agree, this appears to work better than the original JN plugin too !
